__tests__/app/renderer/src/reducers/copiedToClipboard.js

```diff
@@ -34,8 +34,8 @@ describe('copiedToClipboard reducer', () => {
 
   it('should handle the rest of the actions', () => {
     const state = undefined;
-    Object.values(allTypes).map(type =>
```

Comment:
if I remember correctly, this requires `stage-3` to work again. But I wouldn't worry too much about this right. We'll deal with it in conjunction with upgrading `babel`

Reply:
👍